gemrb/vcpkg_deps.cmake

```diff
@@ -29,6 +29,8 @@ ENDIF()
 # libcharset is just a relic of the current iconv build
 LIST(APPEND DLL_SET
 	${VCPKG_DATAROOT}/bin/SDL${DLL_SDL_VER}.dll # Cmake doesn't actually find the sdl debug libs
+	${DLL_DIR}SDL2_mixer.dll
```

Comment:
This should use ${DLL_SDL_VER}, since there's a mixer also for sdl1.

Reply:
I haven't found SDL mixer in VCPkg, is it a part of some other package?